rbac/apps.py

```
@@ -0,0 +1,19 @@
+class RbacConfig(AppConfig):
+    """پیکربندی اپلیکیشن RBAC"""
+    
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'rbac'
+    verbose_name = 'مدیریت نقش‌ها و دسترسی‌ها'
+    
+    def ready(self) -> None:
+        """
+        اجرای کدهای مورد نیاز هنگام بارگذاری اپلیکیشن
+        """
+        # ثبت سیگنال‌ها در صورت نیاز
```

Ruff (0.12.2)

Comment contains ambiguous `ه` (ARABIC LETTER HEH). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF003)